packages/core/.storybook/preview.tsx

```diff
@@ -1,6 +1,5 @@
 import React from "react";
 import * as VibeComponents from "../src/components";
-import * as VibeComponentsNext from "../src/next";
```

Comment:
Let's consider not removing it completely but leaving it as is and just removing the components imports inside

Reply:
Brought it back

Reply:
Moved to line 4